src/components/Reports/ViewPhotos.tsx

```diff
@@ -80,15 +89,16 @@ export default function ViewPhotos({ reportId, onPhotoRemove, editable }: Photos
         prevButtonLabel={strings.PREVIOUS}
         title={strings.PHOTOS}
       />
-      <Box display='flex' flexWrap='wrap'>
+      <Box display='flex' flexWrap='wrap' flexDirection='row'>
         {photos.map((photo, index) => (
           <Box
             key={index}
+            display='flex'
             position='relative'
-            marginRight={theme.spacing(3)}
-            marginTop={theme.spacing(2)}
-            maxWidth='400px'
-            sx={{ cursor: 'pointer' }}
```

Comment:
I think we should keep the cursor pointer to know the photo is clickable

Reply:
oops,, will add it back, had added it into the thumbnail classes instead but that only covers the image and not the enclosing box.